.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "third_party/prjxray-db"]
 	path = third_party/prjxray-db
 	url = https://github.com/SymbiFlow/prjxray-db.git
+[submodule "third_party/cocotb"]
```

Comment:
Does it make sense to just install cocotb like we install pytest?

Reply:
So add it to a conda channel like yosys and vtr? This seems simpler to me.

Reply:
Try `env/conda/bin/pip install cocotb` -- or `env/conda/bin/pip install <git url>` see https://pip.pypa.io/en/stable/reference/pip_install/#git

Reply:
See how I added yapf in #103 pull request.